dev/tools/coqdev.el

```diff
@@ -33,7 +33,7 @@
 
 (defun coqdev-default-directory ()
   "Return the Coq repository containing `default-directory'."
-  (let ((dir (locate-dominating-file default-directory "META.coq")))
+  (let ((dir (locate-dominating-file default-directory "META.coq.in")))
```

Comment:
If that file is meant to leave it may be worth finding something more robust.

Reply:
We could use `dune-project` once it is in the tree.

Reply:
In fact Tuareg should learn to do that automatically.

Reply:
This is coq specific though.

Reply:
Once we use a standard build system I see little coq-specific stuff there.